### PR TITLE
Issue 66 display single agency

### PR DESCRIPTION
**Pull request by seanlawrenz**
Screen Shot 2020-07-27 at 8 42 09 PM
Screen Shot 2020-07-27 at 8 42 57 PM
Screen Shot 2020-07-27 at 6 59 45 PM

### CODE REVIEW

File: src/Modules/Events/AgencyEventListContainer.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import { EventHandler } from '../../Utils/EventHandler';
+import EventListComponent from './EventListComponent';
+
+const AgencyEventListContainer = props => {
+  const { agencyId } = useParams();
+  const [loading, setLoading] = useState(false);
+  const [error, setErrors] = useState(false);
+  const [targetAgency, setTargetAgency] = useState({});
+  const [agencyEvents, setAgencyEvents] = useState([]);
+
```
Comment (davidpickering) [inline src/Modules/Events/AgencyEventListContainer.js:17]: General question. Does any of this content depend upon the user having landed on the main page or search results? We are thinking an agency would like to this page, and there would not be any established states to tap into.

File: src/Modules/Events/EventListComponent.js [hunk 2]
```diff
@@ -35,26 +36,32 @@ const EventListComponent = props => {
...
+                    <span className="day-view-title">
+                      {formatDateDayAndDate(date)}
+                    </span>
+                  </div>
+                </div>
+                <div className="row mt-2">
+                  {event.map(event => (
+                    <EventCardComponent key={event.id} event={event} />
+                  ))}
                 </div>
```
Comment (davidpickering) [inline src/Modules/Events/EventListComponent.js:60]: Not sure exactly how this works, but does this section display dates and then event_dates within a date? From a data perspective, it is possible to have an agency with multiple event_dates/distributions on the same day (e.g. a mobile pantry that operates two distributions on the same day in different counties)

File: src/Modules/Events/AgencyEventListContainer.js (hunk 1 above)
Comment (seanlawrenz) [reply src/Modules/Events/AgencyEventListContainer.js:17]: Good question. We do need the `agencyId` in the params. For example: `http://root/agency/events/123`.

Other than that, the user can go directly to the link. The `useState` is component level only. Meaning redux is not needed.

File: src/Modules/Events/EventListComponent.js (hunk 2 above)
Comment (seanlawrenz) [reply src/Modules/Events/EventListComponent.js:60]: The component will show if there are multiple events within a day.

It's "dumb" enough to not care about dates outside of displaying them together.

**Pull request merged**